django_app/settings.py

```diff
@@ -107,7 +107,11 @@
             'client_id': os.environ.get('GOOGLE_CLIENT_ID'),
             'secret': os.environ.get('GOOGLE_CLIENT_SECRET'),
             'key': ''
-        }
+        },
+        "SCOPE": [
```

Comment:
What's SCOPE mean?

Reply:
Can you add this as a comment?

Reply:
Ok

Reply:
@Programmer-yar What is profile data? Can you point me to an example?

I have a feeling we only really want email. But show me what profile is with a screenshot

Reply:
Profile data include image url, Full name etc. But I agree with you we need email only for now

Reply:
Make an issue and post a link to it here.

Reply:
https://github.com/turian/audio-discrimination-crowdsource/issues/19